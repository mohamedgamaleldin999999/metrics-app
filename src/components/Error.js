import React from 'react';

const Error = () => (
  <>
    <div>
      <span>
        <span style={{ color: 'red' }}>Error! </span>
        Page not Found try
      </span>
    </div>
  </>
);

export default Error;
