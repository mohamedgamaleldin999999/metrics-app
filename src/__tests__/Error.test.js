import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../components/Error';

it('Error renders correctly', () => {
  const navbar = renderer.create(<Error />)
    .toJSON();
  expect(navbar).toMatchSnapshot();
});
