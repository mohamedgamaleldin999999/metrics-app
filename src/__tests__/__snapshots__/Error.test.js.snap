// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<div>
  <span>
    <span
      style={
        Object {
          "color": "red",
        }
      }
    >
      Error! 
    </span>
    Page not Found
  </span>
</div>
`;
